### ISSUE TITLE
[GROUP TEMPLATES] Failed to create group: A value is required for property 'mailNickname' of resource 'Group'.

### ISSUE THREAD

**Issue by tfromcube** [KelvinTegelaar/CIPP#2201]
### Description

1. Go to Administration -> Group Templates
2. Add Group Template 
- Fill in display Name
- Fill in Description
- Leave username blank
- Security group (not mail enabled)
3. Go to Administration -> Deploy Group Template
- Select newly created template, leave everything default (again, leave username blank)
4. Submit
5. Following Error message appears:

![image](https://github.com/KelvinTegelaar/CIPP/assets/153834694/c27a18b7-4301-4da4-a184-c6420bfa1bcb)
This will generate the error **Failed to create group: A value is required for property 'mailNickname' of resource 'Group'.**

Tl;dr group templates can't be deployed unless the username value is filled in.

### Environment data

```PowerShell
Self-hosted instance (But we are sponsors)
Front end version number: Latest: 5.2.1 
Back end version number: Latest: 5.2.1
Tried Tenant Cache Clear: true 
Tried Token Cache Clear: true
```

**Comment by tfromcube** [comment]
I confirm that I have checked the documentation thoroughly and believe this to be an actual bug.

**Comment by KelvinTegelaar** [comment]
Not a bug, you missed filling the username field. Thats mandatory for a security group :)
